presto-prometheus/src/test/java/io/prestosql/plugin/prometheus/MetadataUtil.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.prometheus;
+
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.FromStringDeserializer;
+import com.google.common.collect.ImmutableMap;
+import io.airlift.json.JsonCodec;
+import io.airlift.json.JsonCodecFactory;
+import io.airlift.json.ObjectMapperProvider;
+import io.prestosql.metadata.Metadata;
+import io.prestosql.spi.type.MapType;
+import io.prestosql.spi.type.StandardTypes;
+import io.prestosql.spi.type.Type;
+import io.prestosql.spi.type.TypeManager;
+import io.prestosql.type.InternalTypeManager;
+
+import java.util.Map;
+
+import static io.prestosql.metadata.MetadataManager.createTestMetadataManager;
+import static io.prestosql.spi.type.BigintType.BIGINT;
+import static io.prestosql.spi.type.DoubleType.DOUBLE;
+import static io.prestosql.spi.type.TimestampType.TIMESTAMP;
+import static io.prestosql.spi.type.TypeSignature.mapType;
+import static io.prestosql.spi.type.VarcharType.VARCHAR;
+import static io.prestosql.spi.type.VarcharType.createUnboundedVarcharType;
+import static java.util.Locale.ENGLISH;
+
+public final class MetadataUtil
+{
+    private MetadataUtil()
+    {
+    }
+
+    public static final JsonCodec<PrometheusTable> TABLE_CODEC;
+    public static final JsonCodec<PrometheusColumnHandle> COLUMN_CODEC;
+    public static final JsonCodec<Map<String, Object>> METRIC_CODEC;
+
+    private static final Metadata METADATA = createTestMetadataManager();
+    private static final TypeManager TYPE_MANAGER = new InternalTypeManager(METADATA);
+    static final MapType varcharMapType = (MapType) TYPE_MANAGER.getType(mapType(VARCHAR.getTypeSignature(), VARCHAR.getTypeSignature()));
+
+    static {
```

Comment:
just initialize them in place of declaration. Why do you need this static block?

Reply:
The static block seems like the most concise way to handle a multistep config for the static variables and allowing them to be easily used in the tests.